app/Exceptions/Handler.php

```diff
@@ -17,7 +18,7 @@
 
 class Handler extends ExceptionHandler
 {
-    protected ContextualLogger $logger;
+    // protected ContextualLogger $logger;
```

Comment:
Please, remove commented code and double-check imports

Reply:
ok